databricks/koalas/missing/series.py

```diff
@@ -156,6 +156,11 @@ class _MissingPandasLikeSeries(object):
     real = unsupported_property(
         'real',
         reason="If you want to collect your data as an NumPy array, use 'to_numpy()' instead.")
+    nbytes = unsupported_property(
```

Comment:
@itholic, can you remove `nbytes = unsupported_property('nbytes')` at `_MissingPandasLikeSeries`?

Reply:
Sure, i've fixed it. Thanks !